doc/source/whatsnew/v0.25.0.rst

```diff
@@ -132,7 +132,8 @@ Performance Improvements
 - Improved performance of :meth:`Series.searchsorted`. The speedup is especially large when the dtype is
   int8/int16/int32 and the searched key is within the integer bounds for the dtype (:issue:`22034`)
 - Improved performance of :meth:`pandas.core.groupby.GroupBy.quantile` (:issue:`20405`)
-
+- Improved performance of `tokenize_bytes` in `tokenizer.c`
```

Comment:
also say ``:meth:`read_csv` `` a user has no idea what any of the other things are